modules/core/src/main/java/org/apache/ignite/internal/client/thin/TcpClientCache.java

```diff
@@ -1103,8 +1103,8 @@ public void putAllConflict(Map<? extends K, ? extends T3<? extends V, GridCacheV
      * @param drMap DR map.
      * @return Future.
      */
-    public IgniteClientFuture<Void> putAllConflictAsync(Map<? extends K, T3<? extends V, GridCacheVersion, Long>> drMap)
```

Comment:
why do you need this change?

Reply:
In the first commits I used it in the test for generics. The same parameter set for `#putAllConflictAsync()` and `#putAllConflict()` helped to reduce the boilerplate code. Since I completely changed that part, I think it's no longor neccesary. Changed it back